test/test_helper.rb

```diff
@@ -4,17 +4,17 @@
 ENV["RAILS_ENV"] = "test"
 
 $LOAD_PATH.unshift(File.expand_path("../lib", __dir__))
+ROOT = Pathname.new(__dir__).join("..").expand_path
+
 require "constant_resolver"
 require "packwerk"
 
 require "minitest/autorun"
 require "minitest/focus"
+require "mocha/minitest"
 require "support/test_macro"
 require "support/test_assertions"
-
-require "mocha/minitest"
-
-ROOT = Pathname.new(__dir__).join("..").expand_path
+require "support/application_fixture_helper"
```

Comment:
Is the order of those require statements important or can we sort them alphabetically? If it is maybe it makes sense to add a comment?

Reply:
I think `require` statements are sensitive to ordering in general, since `require` executes the file.  The new `application_fixture_helper` entry is not dependent on the other `support` entries.  I can move it up to follow alphabetical ordering, at least for now.